packages/infrastructure/authentik_github_sso/vars.tf

```diff
@@ -0,0 +1,43 @@
+variable "media_configmap" {
+  description = "The configmap holding the static media that Authentik will use"
+  type        = string
+  default     = "media"
+}
+
+variable "authentik_namespace" {
+  description = "The kubernetes namespace where Authentik is deployed"
+  type        = string
+  default     = "authentik"
+}
+
+variable "authentik_domain" {
+  description = "The domain name of the authentik instance"
+  type        = string
+}
+
+variable "organization_name" {
+  description = "The name of your organization"
+  type        = string
+}
+
+variable "acs_url" {
+  description = "The ACS url provided by Github when configuring an external identity provider"
+  type        = string
+}
+
+variable "audience" {
+  description = "The Entity URL provided by the Service Provider"
```

Comment:
You've said in the docs that this should not have `/sso` in it. I think it'd be appropriate to add a validation here as this seems like something someone could get wrong and then have it not work.